shelley/chain-and-ledger/executable-spec/test/Examples.hs

```diff
@@ -410,6 +420,7 @@ esEx1 = EpochState acntEx1 emptySnapShots lsEx1 ppsEx1
 lastByronHeaderHash :: HashHeader
 lastByronHeaderHash = HashHeader $ unsafeCoerce (hash 0 :: Hash ShortHash Int)
 
+-- | Empty initial state with fake Byron hash and no blocks at all.
```

Comment:
maybe something like "the initial shelley state before any blocks from shelley have been processed"? the blocks are not actually a part of the state...

Reply:
To avoid expanding long sentence, I added another: 
"No blocks of Shelley have been processed yet"?